src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -399,6 +220,14 @@ static inline int MPIDI_OFI_do_put(const void *origin_addr,
         goto fn_exit;
     }
 
+    if (origin_density < MPIR_CVAR_CH4_IOV_DENSITY_MIN &&
+        target_density > MPIR_CVAR_CH4_IOV_DENSITY_MIN) {
```

Comment:
```suggestion
        target_density >= MPIR_CVAR_CH4_IOV_DENSITY_MIN) {
```

Reply:
FYI, this error occurs in more places too.  So, please search for them and replace all of them.